### DISCUSSION TITLE
How to preserve sqlite database file across builds?

### DISCUSSION THREAD

**Discussion by B-Esmaili** [vercel/vercel#5327] [Help] [answered]
I am using prisma to connect to `sqlite` database in vercel platform but it looks like in every deploy vercel cleans up the directory and regenerate production files. Is there any way to instruct vercel to preserve sqlite database file between incremental deploys?

**Comment by mcsdevv** [comment, marked as answer]
Hello @B-Esmaili, Vercel is a serverless platform and so all state is lost between Serverless Function executions.

If you require state to be persisted, you will need to use a database hosted externally: https://vercel.com/docs/solutions/databases

**Comment by B-Esmaili** [comment]
@mcsdevv Thanks for clarifications.
